### PR TITLE
Windows: Build load.go

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com
@swernli

This PR is part of the proposal described in issue 10662 to port the docker daemon to Windows. This PR ensures graph\load.go is compiled in for the Windows daemon, factoring out DEFAULTTAG and ValidateTagName to the pkg\tags directory to avoid the Windows client does not attempt to pull in the entire graph driver.

### CODE REVIEW

**Comment by tiborvass** [comment]
@jhowardmsft i feel it is too Docker dependent to put it in `pkg`, I suggest `graph/tags` instead.

**Comment by jhowardmsft** [comment]
Fair comment. Will update

**Comment by jhowardmsft** [comment]
That's moved now.

**Comment by tiborvass** [comment]
LGTM

File: graph/manifest_test.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 
 	"github.com/docker/docker/image"
+	"github.com/docker/docker/pkg/tags"
```
Comment (tiborvass) [inline graph/manifest_test.go]: @jhowardmsft relic from previous version

Comment (jhowardmsft) [inline graph/manifest_test.go]: Good catch, thanks. Fixed.

File: graph/manifest_test.go [hunk 2]
```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 	"testing"
 
+	"github.com/docker/docker/graph/tags"
```
Comment (tiborvass) [inline graph/manifest_test.go]: @jhowardmsft not used

Comment (jhowardmsft) [inline graph/manifest_test.go]: Fixed

File: graph/tags/tags_unit_test.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package tags
+
+import (
+	"testing"
+
+	"github.com/docker/docker/graph/tags"
```
Comment (LK4D4) [inline graph/tags/tags_unit_test.go]: you importing same package. you don't need it.

Comment (jhowardmsft) [inline graph/tags/tags_unit_test.go]: Fixed. Thanks for catching.

**Comment by cpuguy83** [comment]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**